allensdk/brain_observatory/ecephys/stimulus_analysis/stimulus_analysis.py

```diff
@@ -127,8 +154,10 @@ def stim_table(self):
             )
 
             if self._stim_table.empty:
-                # stim_names = self._stimulus_names or self._stimulus_key
-                raise Exception(f'Could not find stimulus data with presentation name {self._stimulus_key}')
+                raise Exception(f'Could not find stimulus data with "stimulus_key" {self._stimulus_key}')
+
+            # TODO: Should we remove columns that are not relevant to the selected stimulus? If a feature for another
```

Comment:
not a review comment per se, but this seems worth following up on. I think the answer is that get_presentations_for_stimulus should do a better job yanking irrelevant params.